styles/designs/carnival/parts/_lists-shapes.scss

```diff
@@ -76,3 +76,61 @@
         )),
     )
 ));
+
+@include create_shape('carnival:::ExercisesListsShape',(
+    _components: (
+        map-merge($ListContainer,(
+            _components: (
+                map-merge($DivWrapper--FirstLevel,(
+                    _components: (
+                      $UnorderedList--FirstLevel--Exercise,
+                      map-merge($OrderedList--FirstLevel--Exercise, (
+                        _components: (
+                          map-merge($OrderedList--SecondLevel--Exercise, (
+                            _components: (
+                              $OrderedList--SecondLevel--Exercise,
+                            )
+                          )),
+                        )
+                      )),
+                    )
+                )),
+                map-merge($DivWrapper--SecondLevel, (
+                  _components: (
+                    $UnorderedList--FirstLevel--Exercise,
+                    map-merge($OrderedList--FirstLevel--Exercise, (
+                        _components: (
+                          $OrderedList--SecondLevel--Exercise
+                        )
+                    )),
+                  )
+                )),
+              )
+        )),
+    )
+));
+
+// Composite Page List
+@include create_shape('carnival:::ExerciseCompositePageListsShape',(
+    _components: (
+        map-merge($ListContainer,(
+            _components: (
+                map-merge($DivWrapper--FirstLevel,(
+                    _components: (
+                      map-merge($OrderedList--FirstLevel--Exercise, (
+                        _components: (
+                          $OrderedList--SecondLevel--Exercise,
+                        )
+                      )),
+                    )
+                )),
+                map-merge($SectionWrapper--FirstLevel,(
+                    _components: (
+                      $UnorderedList--FirstLevel--Exercise,
+                      $OrderedList--FirstLevel--Exercise,
+                    )
+                )),
+            )
+        )),
+    )
+));
```

Comment:
I have created a separate shape for lists present in eoc exercise modules (they are outside exercise) to avoid conflicts and redundancy with Natalia's work on refactoring eoc.